src/tests/bdd/document/reference.feature

```diff
@@ -162,7 +161,8 @@ Feature: Add and remove references
     {
       "name": "myMooring",
       "type": "dmss://test-DS/TestData/Mooring",
-      "_id": "3f9ff99f-9cb5-4afc-947b-a3224eee341f"
+      "_id": "3f9ff99f-9cb5-4afc-947b-a3224eee341f",
+      "Bigness": 1
```

Comment:
This is wrong. If the validator forces us to send complete entites, when only inserting references, that must be fixed somehow.